src/PSDocs.Azure/docs/Azure.Conventions.Doc.ps1

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Synopsis: Name documents by parent path.
+Export-PSDocumentConvention 'Azure.NameByParentPath' {
+    $template = Get-Item -Path $PSDocs.TargetObject;
+
+    # Get parent directory paths where <name>/v<version>/template.json
+    $version = $template.Directory.Name;
+    $name = $template.Directory.Parent.Name;
+
+    # Name the document <name>_<version>.md
+    $docName = $version;
+    if ($version -like 'v*') {
```

Comment:
See above comment - given this comparison, might want to stipulate that start with "v".